### ISSUE TITLE
Explorer hover text can become sticky when primary side bar is positioned on the right

### ISSUE THREAD

**Issue by gjsjohnmurray** [microsoft/vscode#174547]
Type: <b>Bug</b>

I will attach a GIF

VS Code version: Code - Insiders 1.76.0-insider (dc619de8d9efdabd781fc5658d0051813c347055, 2023-02-16T05:23:44.671Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: Yes

**Comment by gjsjohnmurray** [comment]
Latest Insiders defaults to old style hover tips. 👍 

> Version: 1.76.0-insider (user setup)
> Commit: 96a795cc0b1cd62f9ff66c347637f917c6dde3da
> Date: 2023-02-17T22:22:55.538Z

After re-enabling the new style ones:
> `"explorer.experimental.hover": true`

I am pleased that the click-through and mousewheel-scroll-through issues reported elsewhere seem to have been fixed.

This shift-left issue remains, and the displaced tip now mostly disappears under the editor:

![image](https://user-images.githubusercontent.com/6726799/219855038-c2c4fe38-67b8-4ce2-9109-e95b7d721c41.png)

**Comment by lramos15** [comment]
> This shift-left issue remains, and the displaced tip now mostly disappears under the editor:

Yeah that's why we're not ready to ship this just yet. It seems like the fix for scroll and mouse click (which involves making the hover a child element of the list item) causes the hover to get bound to the list making it useless for long file names. You can see this even on the left hand side with a long file name. The whole text won't be visible.
